.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ executors:
           MB_DB_DBNAME: metabase_test
           MB_DB_USER: metabase_test
           MB_POSTGRESQL_TEST_USER: metabase_test
-      - image: circleci/postgres:latest
+      - image: cimg/postgres:15.0
```

Comment:
AFAIK, Metabase is not compatible with Postgres 15 (I had to downgrade my container when I've done the upgrade to 0.44.6 few days ago)

Reply:
hi @mickaelandrieu can you post a new issue about what you saw? Metabase should be compatible, in fact I've been running Metabase with PG15 for a few days without any issue

Reply:
@paoliniluis I'd prefer leaving this on Postgres latest so if a new version (like 15) breaks we find out sooner rather than later. If CircleCI doesn't have a `latest` image anymore we can just use the official Postgres image... there's no reason we need to use the `circleci/` or `cimg/` versions of images. The only reason we did in the first place was that they were supposed to be cached and load faster. But I think that was a lie. It's slow either way